wdl/transforms/new-base/src/main/scala/wdl/transforms/base/ast2wdlom/AstToCallElement.scala

```diff
@@ -30,8 +30,15 @@ object AstToCallElement {
 
     val callBodyValidation: ErrorOr[Option[CallBodyElement]] = ast.getAttributeAsOptional[CallBodyElement]("body").toValidated
 
-    (callableNameValidation, aliasValidation, afterValidation, callBodyValidation) mapN { (name, alias, after, body) =>
-      CallElement(name, alias, after, body)
+    val srcLocValidation : ErrorOr[Option[SourceFileLocation]] = ast.getSourceLine match {
+      case None =>
+        None.validNel
+      case Some((startLine)) =>
+        Some(SourceFileLocation(startLine)).validNel
```

Comment:
I think you could save some repetitive code by creating a companion object static function for `SourceFileLocation` that takes in an `Option[Int]` and returns a `Option[SourceFileLocation]`